### PR TITLE
Add Countdown Flipclock

### PR DESCRIPTION
**Pull request by danrabbit**
- [x] Set the correct time 
- [x] Figure out why jquery isn't loading 
- [x] Load countdown.css in countdown.php template

### CODE REVIEW

File: scripts/countdown.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var clock = $('.countdown').FlipClock({
```
Comment (lewisgoddard) [inline scripts/countdown.js]: This should all be wrapped in a jQuery `onload`.

File: _templates/countdown.php [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha256-cCueBR6CsyA4/9szpPfrX3s49M9vUU5BgtiJj06wt/s=" crossorigin="anonymous"></script>
```
Comment (lewisgoddard) [inline _templates/countdown.php]: I don't understand what you're doing with the script tags here, jQuery should already be loaded by the page anyway, why are we including it again?

File: _templates/countdown.php [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+<div class="countdown-wrapper">
+    <div class="countdown"></div>
+    <a class="read-more" href="#">Click to Continue</a>
+</div>
+
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/flipclock/0.7.8/flipclock.min.js"></script>
```
Comment (lewisgoddard) [inline _templates/countdown.php]: We're mixing CDNs, we'd be better off updating jsDelivr and loading from there.

File: _templates/countdown.php (hunk 2 above)
Comment (danrabbit) [inline _templates/countdown.php]: Yeah I noted this in the PR comments. For some reason jquery isn't being loaded until later, maybe. I dunno. It's not working though.

Comment (lewisgoddard) [inline _templates/countdown.php]: We load it asynchronously, so you need to wrap `scripts/countdown.js` with the usual jQuery onload parameter that only gets fired once it's there.

File: _templates/header.php [hunk 4]
```diff
@@ -100,6 +100,7 @@
         <script>
             <?php include __DIR__.'/../scripts/jql.min.js'; ?>
             jQl.loadjQ('https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.0/jquery.min.js')
+            jQl.loadjQdep('https://cdnjs.cloudflare.com/ajax/libs/flipclock/0.7.8/flipclock.min.js')
```
Comment (lewisgoddard) [inline _templates/header.php]: Could this not be moved to the `countdown.php`, keep it neater? These dependencies don't have to be loaded together.

Comment (btkostner) [inline _templates/header.php]: I was having some trouble with it being called in the countdown.

- If it's after the `boot` call then it won't work as intended
- If the `boot` call is put in the footer then there is a chance the other scripts will load and jQl won't intercept the `ready` calls
- If it's a regular (non async) call then it won't always load after jQuery is loaded and give an undefined error

Comment (btkostner) [inline _templates/header.php]: Once this is merged with master, we can update the loki branch to take it out so on release we are all clean.

**Comment by lewisgoddard** [comment]
I hereby approve this code.

File: _templates/header.php (hunk 4 above)
Comment (lewisgoddard) [inline _templates/header.php]: I've made a pull request for flipclock to be auto-updated on JSDelivr: https://github.com/jsdelivr/jsdelivr/pull/13425

**Comment by lewisgoddard** [comment]
This now has conflicts, and we might be able to remove a lot of the CSS as I have asked JSDelivr to host it.

**Comment by danrabbit** [comment]
We have a date. September 5th. So as soon as we get this sorted it can be put up.

**Comment by danrabbit** [comment]
- [ ] Can we add a "Days" section or would that be too much work?
- [ ] Do we know what timezone this is set to? Is it UTC? We need it to be on the same timezone no matter who's viewing so it ends a the right time for everyone

**Comment by btkostner** [comment]
The countdown on it's own is a little vague for someone who is first visiting the site. We should include some hype text above it so people know what we are counting down to.

- hype text

**Comment by danrabbit** [comment]
It's intentionally vague. We never include text indicating what it's for

**Comment by lewisgoddard** [comment]
https://github.com/jsdelivr/jsdelivr/pull/13425 being merged will neaten this up.

**Comment by lewisgoddard** [comment]
I'm not sure if we have modified it here at all, but there's now a CDN hosted version of `flipclock.css` at https://cdn.jsdelivr.net/g/flipclock@0.7.7(flipclock.css)

**Comment by btkostner** [comment]
When we push this to master, we should push translations for #1157 and hopefully get most of the content translated before release.

- [ ] Push Loki branch translations

**Pull request merged**